pkg/container-hook/tracer.go

```diff
@@ -270,13 +270,16 @@ func (n *ContainerNotifier) install() error {
 	if runtimePath != "" {
 		log.Debugf("container-hook: trying runtime from RUNTIME_PATH env variable at %s", runtimePath)
 
-		if _, err := os.Stat(runtimePath); errors.Is(err, os.ErrNotExist) {
+		// Resolve symlinks
+		realPath, err := host.EvalSymlinks(runtimePath)
```

Comment:
I think it is better to use the following:
```
HOST_ROOT=/host
RUNTIME_PATH=/var/lib/rancher/k3s/data/current/bin/runc
```
And then use:
```
securejoin.SecureJoin(host.HostRoot, runtimePath)
```

It does resolve the symlinks within the specified root.

See: https://github.com/cyphar/filepath-securejoin

@i-Pear is currently adding securejoin in the [uprobe PR #2634](https://github.com/inspektor-gadget/inspektor-gadget/pull/2634/commits/6117688cf74d370a536cd993d079121201a1c24d#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6).

Reply:
nice, let me check that later if that works for our customers, then I will revisit this PR
thanks @alban

Reply:
I need to check if that works for us...

Reply:
@alban it works with the caveat that we should remove `/host` to the existing configs
don't worry about that, we will document in our release notes